fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -723,7 +743,7 @@ private void refreshExternalTable(TaskRunContext context,
             // refresh old table
             Table table = optTable.get();
             if (table.isNativeTableOrMaterializedView() || table.isConnectorView()) {
```

Comment:
what's the difference between a ConnectorView and a View?  it is a view created in StarRocks or in Hive/Iceberg natively?